relay-cabi/src/processing.rs

```diff
@@ -151,6 +155,8 @@ pub unsafe extern "C" fn relay_pii_strip_event(
     let mut processor = PiiProcessor::new(&compiled);
 
     let mut event = Annotated::<Event>::from_json((*event).as_str())?;
+    validate_annotated_transaction(&mut event)?;
+    light_normalize_event(&mut event, None, None, None, None, None, None)?;
```

Comment:
Why add light normalization here? `processor` at this point is a `PiiProcessor`, not a `StoreProcessor`, so the call to `process_value` is not related to store normalization.